invenio_rdm_records/templates/semantic-ui/invenio_rdm_records/oai-details.html

```diff
@@ -0,0 +1,11 @@
+{#
+  Copyright (C) 2020 CERN.
+
+  Invenio App RDM is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+{% extends "invenio_administration/details.html" %}
+
+{% block javascript %}
+  {{ webpack['invenio-oai-pmh-details.js'] }}
```

Comment:
In terms of jinja, we don't need to override anything. The overridable component is provided by `invenio-administration`. 

Here we only need custom javascript to either override the component completely or render additional components.